docs/source/getting-started/preparing-to-install.adoc

```diff
@@ -0,0 +1,61 @@
+include::variables.adoc[]
+
+[[installing-minishift]]
+= Preparing to Install {project}
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[installing-overview]]
+== Overview
+
+The following section describes how to install {project} and the required dependencies.
+
+The {project} setup procedure should be run as a regular user with permission to launch virtual machines.
+In the procedure, you will see how to assign that permission, along with ways to configure your hypervisor and command shell to start and effectively interact with {project}.
+
+[[install-prerequisites]]
+== Prerequisites
+
+{project} requires a hypervisor to start the virtual machine on which the OpenShift cluster is provisioned.
+Verify that the hypervisor of your choice is installed and enabled on your system before you set up {project}.
+Once the hypervisor is up and running, additional setup is required for {project} to work with that hypervisor.
+
+Depending on your host operating system, you have the choice of the following recommended native hypervisors:
+
+macOS::
+- link:https://github.com/mist64/xhyve[xhyve]
+
+Linux::
+- link:https://en.wikipedia.org/wiki/Kernel-based_Virtual_Machine[KVM]
+
+Windows::
+- link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V]
++
+[NOTE]
+====
+To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] using the Hyper-V Manager and set the environment variable `HYPERV_VIRTUAL_SWITCH` to this virtual switch.
+For specific configuration steps see the xref:../getting-started/setting-up-virtualization-environment.adoc#[Setting Up the Virtualization Environment] section.
+====
+
+link:https://www.virtualbox.org/wiki/Downloads[VirtualBox] may also be used for all supported operating systems instead of the recommended native hypervisor.
```

Comment:
it is missing the bullet point. 
<img width="845" alt="screen shot 2018-03-26 at 5 56 51 pm" src="https://user-images.githubusercontent.com/3125299/37906272-209bdd5e-311f-11e8-91f3-5253085adf66.png">

Reply:
It's not a part of the definition list, as VirtualBox applies to all platforms -- adding a bullet point may lead to the user thinking that it's only supported on Windows.

We could add another item to the definition list to contain VirtualBox, though, perhaps "All Platforms" or something similar?